src/v1/index.d.ts

```diff
@@ -0,0 +1,17 @@
+/*!
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export {DatastoreClient} from './datastore_client';
```

Comment:
🙋 it looks like this overlaps with the proto method of generating these in #305?

Reply:
Yeah, I didn't even notice that PR existed until I submitted this one, lol. I think there's still value in this PR since #305 does not supply types to the gapics, although it would probably be beneficial to wait for it to land and re-use the interfaces instead of having similar ones/duplicates.